Lab9.py

```diff
@@ -0,0 +1,62 @@
+class Animals:
```

Comment:
Firstly, class names should be singular: Animal, Bird, Mammal etc.
Secondly, I believe your Animal class is meant to be abstract, and inherit [abc.ABC](https://docs.python.org/3/library/abc.html).